tsl/src/continuous_aggs/create.c

```diff
@@ -368,6 +370,39 @@ cagg_add_trigger_hypertable(Oid relid, char *trigarg)
 	if (check_trigger_exists_hypertable(relid, CAGGINVAL_TRIGGER_NAME))
 		return;
 	ht = ts_hypertable_cache_get_cache_and_entry(relid, CACHE_FLAG_NONE, &hcache);
+	if (hypertable_is_distributed(ht))
+	{
+		DistCmdResult *result;
+		List *data_node_list = ts_hypertable_get_data_node_name_list(ht);
+		const char **cmds; /* same order as ht->data_nodes */
+		ListCell *cell;
+
+		int i = 0;
+		cmds = palloc(list_length(data_node_list) * sizeof(*cmds));
+		foreach (cell, ht->data_nodes)
+		{
+			HypertableDataNode *node = lfirst(cell);
+			StringInfo command = makeStringInfo();
+			appendStringInfo(command,
+							"CREATE TRIGGER %s AFTER INSERT OR UPDATE OR DELETE ON %s.%s FOR EACH ROW EXECUTE FUNCTION %s.%s(%d)",
+							CAGGINVAL_TRIGGER_NAME,
```

Comment:
Probably good to quote this as well so that it work even with strange trigger names (not likely to happen, just a precaution).
```suggestion
							quote_identifier(CAGGINVAL_TRIGGER_NAME),
```